### PR TITLE
Fixed create migration, added figgy config initializer, added rvm .dotfiles

### PR DESCRIPTION
**Pull request by enovajuan**
Fixed landable migration file to test for extension existance, which can break builds. Added dotfiles for rvm and gemset configurations.

*warning*
Added figgy for configuration files. However was not able to get tests to run.
*warning*

Module [landable]:
  * install_generator.rb
  + app_config.rb
  * landable.rb
  + landable.yml
  * 20130510221424_create_landable_schema.rb
  * landable.rb
  + .ruby-gemset
  + .rvmrc
  * landable.gemspec

### CODE REVIEW

File: .ruby-gemset [hunk 1]
```diff
@@ -0,0 +1 @@
+enova_landable
```
Comment (dkaplan88) [inline .ruby-gemset]: we try and keep this as enovaless as possible due to its opensourceness, can we just make this landable?

File: lib/landable/version.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 module Landable
   module VERSION
     MAJOR = 1
-    MINOR = 12
-    PATCH = 1
+    MINOR = 13
```
Comment (dkaplan88) [inline lib/landable/version.rb]: Dont do version bumps in PRs.  Do update the changelog

When we are ready to do a release to ruby gems we do all in one commit to master..

**Comment by dkaplan88** [comment]
In general, I am unsure about figgy as since landable is an engine, configuration I feel like should be a responsibility of the parent application.

I am unsure, though, because I like how it enables abstraction of the landable config.  But figgy then becomes a dependent

**Comment by avavita** [comment]
We already implemented figgy.

**Pull request closed without merging**